api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -33,11 +34,13 @@
 	private String name;
 	
 	private String description;
-	
+
 	private Set<Integer> memberIds;
```

Comment:
@wluyima I missed out on this. I haven't come around to remove this field, because it errors out with the getters and setters saying: `cannot find symbol`. Do I repurpose the getters and setters to something like this:

```
	public Set<Integer> getMemberIds() {
		Set<Integer> memberIds = new TreeSet<Integer>();
		if (CollectionUtils.isNotEmpty(getMembers())) {
			for (CohortMembership member : getMembers()) {
				memberIds.add(member.getCohortMemberId());
			}
		}
		return memberIds;
	}

	public void setMemberIds(Set<Integer> memberIds) {
		memberIds = new TreeSet<Integer>(memberIds);
	}
```